examples/AdvancedOperations/AddSmartDisplayAd.cs

```diff
@@ -0,0 +1,372 @@
+                customerId.ToString(), new CampaignBudgetOperation[] { budgetOperation });
+
+            string budgetResourceName = response.Results.First().ResourceName;
+
+            // Print out some information about the added ad group.
+            Console.WriteLine($"Added campaign budget with resource name = '{budgetResourceName}'.");
+
+            return budgetResourceName;
+        }
+
+        /// <summary>
+        /// Creates the smart display campaign.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>
+        /// <param name="budgetResourceName">The campaign budget resource name.</param>
+        /// <returns>Resource name of the newly created campaign.</returns>
+        private string CreateSmartDisplayCampaign(GoogleAdsClient client, long customerId,
+            string budgetResourceName)
+        {
+            // Get the CampaignService.
+            CampaignServiceClient campaignService = client.GetService(Services.V2.CampaignService);
+
+            // Create the campaign.
+            Campaign campaign = new Campaign()
+            {
+                Name = "Smart Display Campaign #" + ExampleUtilities.GetRandomString(),
+
+                // Smart Display campaign requires the advertising_channel_type as 'DISPLAY'.
+                AdvertisingChannelType = AdvertisingChannelType.Display,
+
+                // Smart Display campaign requires the advertising_channel_sub_type as
+                // 'DISPLAY_SMART_CAMPAIGN'.
+                AdvertisingChannelSubType = AdvertisingChannelSubType.DisplaySmartCampaign,
+
+                // Recommendation: Set the campaign to PAUSED when creating it to prevent
+                // the ads from immediately serving. Set to ENABLED once you've added
+                // targeting and the ads are ready to serve
+                Status = CampaignStatus.Paused,
+
+                // Smart Display campaign requires the TargetCpa bidding strategy.
+                TargetCpa = new TargetCpa() { TargetCpaMicros = 5000000 },
+
+                CampaignBudget = budgetResourceName,
+
+                // Optional: Set the start date.
+                StartDate = DateTime.Now.AddDays(1).ToString("yyyyMMdd"),
+
+                // Optional: Set the end date.
+                EndDate = DateTime.Now.AddYears(1).ToString("yyyyMMdd"),
+            };
+
+            // Create the operation.
+            CampaignOperation operation = new CampaignOperation() { Create = campaign };
+
+            // Add the campaign.
+            MutateCampaignsResponse response = campaignService.MutateCampaigns(
+                customerId.ToString(), new[] { operation });
+
+            string campaignResourceName = response.Results.First().ResourceName;
+
+            // Print out some information about the added ad group.
```

Comment:
I think this should be campaign and not ad group based on the context.

Reply:
Done everywhere.